### DISCUSSION TITLE
blog/zh/how-to-integrate-umami-for-free-to-blog-site/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#5] [Announcements]
# blog/zh/how-to-integrate-umami-for-free-to-blog-site/

Ladder@本篇文章介绍如何零成本给博客或者网站集成 umami 统计功能。数据库用的是 supabase 提供的有限额的 postgres，不过免费提供的 500M 对于  umami 来讲已经足够了。
托管 umami 服务用的是 vercel。得益于现在的云厂商优秀的服务能力，你可以在 10 分钟内集成好 umami。可以点击 数据统计看板 查看最终效果。

https://guangzhengli.com/blog/zh/how-to-integrate-umami-for-free-to-blog-site/

**Comment by Angiin** [comment]
目前我的博客（https://inkdata.net）使用的是 Clicky （https://clicky.com/），Umami 看着确实也很美丽。考虑要不要给我的博客也接入 Umami。

**Comment by naodeng** [comment]
Supabase的 PgBouncer connection string在 1 月15号就停用了，要使用new Supavisor connection string

![URkNzH](https://cdn.jsdelivr.net/gh/naodeng/blogimg@master/uPic/URkNzH.png)

**Comment by chelseachen007** [comment]
血泪教训，数据库密码不要带有@符号，会导致数据库连接不上，排查好久

**Comment by HungerDeng** [comment]
请教下，你的网站流量Referrers里40%+流量都是来自None的，None代表什么呢？

**Reply by guangzhengli** [reply]
代表直接访问或者统计不到来源的流量，比如 RSS 或者AdBlocker 过滤的
